From de9509ae43b7892d4181e1547e2e219d7adb9b0e Mon Sep 17 00:00:00 2001
From: Carsten Haitzler (Raster <raster@rasterman.com>
Date: Thu, 30 Oct 2008 16:06:52 +1100
Subject: [PATCH] exquisitie: update initial timeout to wait for exquisite and the total boot too

this increases the initial timeout wait from 20 to 40 seconds for exqusitie
and also extends the total boot time timeout to 300 seconds from 180, incase
the boot takes a long time.
---
 packages/e17/exquisite/exquisite-init |    6 +++---
 packages/e17/exquisite_svn.bb         |    2 +-
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/packages/e17/exquisite/exquisite-init b/packages/e17/exquisite/exquisite-init
index 6c94fdc..1750184 100644
--- a/packages/e17/exquisite/exquisite-init
+++ b/packages/e17/exquisite/exquisite-init
@@ -33,8 +33,8 @@ export EXQUISITE_IPC=$EXDIR/exquisite
 mount tmpfs -t tmpfs $EXDIR -o,size=40k
 exquisite -fb -fs $args &
 
-exquisite-write -wait 20
+exquisite-write -wait 40
 exquisite-write "PULSATE"
 
-# kill off exquisite in 180 seconds if its not gone yet
-(sleep 180; exquisite-write "QUIT")&
+# kill off exquisite in 300 seconds if its not gone yet
+(sleep 300; exquisite-write "QUIT")&
diff --git a/packages/e17/exquisite_svn.bb b/packages/e17/exquisite_svn.bb
index f1c8908..e303600 100644
--- a/packages/e17/exquisite_svn.bb
+++ b/packages/e17/exquisite_svn.bb
@@ -2,7 +2,7 @@ DESCRIPTION = "A psplash replacement for display"
 LICENSE = "MIT BSD"
 DEPENDS = "eet evas ecore embryo edje"
 PV = "0.0.1+svnr${SRCREV}"
-PR = "r2"
+PR = "r3"
 RDEPENDS = "initscripts"
 RRECOMMENDS_${PN} = "exquisite-themes"
 
-- 
1.5.4.3

